GOVERNANCE.md

```diff
@@ -0,0 +1,153 @@
+# Eiffel Community Governance
+
+Eiffel Community initiated the work to establish governance and put the necessary structures in place in order to have a healthy, sustainable,
+and attractive community.
+
+The community aims to adapt and follow the practices employed by other successful open source communities with minor adjustments based on
```

Comment:
Maybe: 
"The Eiffel community proactively adopts and follows best practices employed by successful open source communities while maintaining a governance agreeable to it's members."

Reply:
will amend